tests/test_add_meta_tag.py

```diff
@@ -10,7 +10,10 @@ class TestAddMetaTag(unittest.TestCase):
 
     def setUp(self):
         if not os.path.exists("/opt/snare/pages/test"):
```

Comment:
this part is completely similar for most of the tests. We can think about separating it.
And instead of test folder to avoid using flags we can generate random string, it also might have collisions, but the probability is near to 0